src/test/java/com/arpnetworking/metrics/impl/TsdTimerTest.java

```diff
@@ -166,8 +168,7 @@ public void testGetElapsedAfterStop() {
         Mockito.verify(logger, Mockito.never()).warn(MockitoHamcrest.argThat(Matchers.any(String.class)));
         timer.getValue();
         Mockito.verify(logger).warn(MockitoHamcrest.argThat(Matchers.any(String.class)));
-        timer.getUnit();
-        Mockito.verify(logger, Mockito.times(2)).warn(MockitoHamcrest.argThat(Matchers.any(String.class)));
+        Mockito.verify(logger, Mockito.times(1)).warn(MockitoHamcrest.argThat(Matchers.any(String.class)));
```

Comment:
[nit] This looks like a restatement (or slightly stronger version of) the previous line?

Reply:
That was left over statement when removing the call to `getUnit`; both would have emitted a warning.